x11-misc/polybar/polybar-3.4.3.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+XPP_COMMIT="044e69d05db7f89339bda1ccd1efe0263b01c8f6"
```

Comment:
What are these for?

Reply:
Ah, they are no longer needed because I changed src_uri to tar package which already has these.  
Removing these.

Reply:
done